src/shared/pages/team/teamView/teamView.tsx

```diff
@@ -125,7 +145,7 @@ const TeamView: React.FC<TeamMembersProps> = ({ history, match, hasRole }) => {
                 }
                 <button type="submit" className="govuk-button govuk-!-margin-right-1 add-team-members-button" data-module="govuk-button" onClick={() => onAddTeamMembersAddClick(history, teamId as string)}>Add team members</button>
                 <button type="submit" className="govuk-button govuk-!-margin-right-1 govuk-button--secondary manage-nominated-contacts-button" data-module="govuk-button" onClick={() => onAddNominatedContactClick(history, teamId as string)}>Manage nominated contacts</button>
-                {hasRole('RENAME_TEAM') &&
+                {(hasRole('RENAME_TEAM') || hasRole('REASSIGN_TEAM_UNIT')) &&
```

Comment:
I wonder if it's worth creating a `hasOneOfRoles(['RENAME_TEAM', 'REASSIGN_TEAM_UNIT')` function (and then use `Array.prototype.includes()` or `Array.prototype.some()`

Reply:
I have made this change